fmriprep/workflows/base.py

```diff
@@ -222,16 +222,15 @@ def init_single_subject_wf(subject_id):
         freesurfer=config.workflow.run_reconall,
         hires=config.workflow.hires,
         longitudinal=config.workflow.longitudinal,
-        num_t1w=len(subject_data['t1w']),
         omp_nthreads=config.nipype.omp_nthreads,
         output_dir=str(config.execution.output_dir),
         reportlets_dir=reportlets_dir,
         skull_strip_fixed_seed=config.workflow.skull_strip_fixed_seed,
-        # skull_strip_mode='force',
+        skull_strip_mode='force',
```

Comment:
surely this will have a config option?

Reply:
yes (we are talking about that right now in #1829 )